views/membership.erb

```diff
@@ -0,0 +1,261 @@
+<div class="hero">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6">
+        <h1><a href="/clenstvo">Staňte sa naším <strong>členom</strong> a budte aj vy Slovensko.digital</a></h1>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="section main-content">
+  <div class="container">
+    <div class="row">
+      <div class="sidebar col-md-3">
+        <ul class="sidebar-menu list-unstyled">
+          <li><a href="/podpora">Podporte finančne</a></li>
+          <li><a href="/venujte-svoj-cas">Venujte svoj čas</a></li>
+          <li class="sidebar-menu-item-active">Členstvo</li>
+        </ul>
+      </div>
+      <div class="col-md-9">
+        <h1 class="content-heading text-center">Prečo byť členom?</h1>
+        <p class="lead"> “Prečo sa stať členom?” Lebo spolu, ako komunita, dosiahneme viac. Tvoríme alianciu firiem a jednotlivcov, ktorým záleží na lepších službách štátu a budúcnosti tejto krajiny. Sme hrdí na to, že podnikáme a pracujeme poctivo a spájame ľudí, ktorí to vidia rovnako.</p>
+
+        <p class="lead">Pridajte sa k nám a získate:</p>
+          <ul>
+            <li><p>Možnosť rozhodovať o budúcnosti Slovensko.Digital</p></li>
+            <li><p>Network, komunitu a výmenú skúseností</p></li>
+            <li><p>Informácie o dianí v štátnom IT - členský newsletter s aktualitami a príležitosťami</p></li>
+            <li><p>Propagáciu v IT komunite a na verejnosti</p></li>
+            <li><p>Bezplatné služby pre Vaše podnikanie</p></li>
+          </ul>
+
+        <p class="lead"><a href="/clenovia">Zoznam členov</a></p>
```

Comment:
Dajme toto tiez bez `lead` a mozno preformulovat na daco ako 'Pozrite si zoznam nasich clenov'. Lebo momentalne to vyzera ako nadpis.

![image](https://user-images.githubusercontent.com/1374745/31383874-efc5b6d6-adbc-11e7-9fbb-deb6001bc303.png)